frontend/src/components/core/StreamlitDialog/StreamlitDialog.tsx

```diff
@@ -219,12 +219,12 @@ function clearCacheDialog(props: ClearCacheProps): ReactElement {
             </div>
           </ModalBody>
           <ModalFooter>
-            <ModalButton kind={Kind.SECONDARY} onClick={props.onClose}>
+            <ModalButton kind={Kind.TERTIARY} onClick={props.onClose}>
```

Comment:
Maybe it would be worth investigating if we could add an e2e test for the Streamlit Dialog. The main reason would be that currently, we don't have any test covering the UI of the `TERTIARY` button or?

Reply:
That's true, for now added FE snapshot tests for Secondary and Tertiary dialog buttons that cover the css 👍🏼